docker/openproblems-r-base/Dockerfile

```diff
@@ -1,5 +1,21 @@
 FROM singlecellopenproblems/openproblems:latest
 
+ARG NB_USER="sagemaker-user"
+ARG NB_UID="1000"
+ARG NB_GID="100"
```

Comment:
Optional improvement: stack these with semicolons, so that you have fewer docker layers.

Reply:
Can you show an example of this?

Reply:
Just read up, `ARG` is the only thing you can't stack... I just assumed it could be done like `ENV`.